### PR TITLE
always print json format when json format flag is specified

### PR DESCRIPTION
**Pull request by devinsag**
update output for json dev empty to be empty set

---
### Edit

Tested just to be sure the changes will only affect output when using  json format flag specifically.
```
architect-local dev:list --format JSON
{}
architect-local dev:list --format json
{}
architect-local dev:list --format table
There are no active dev instances yet. Use `architect dev` to create one.
architect-local dev:list --format TABLE
There are no active dev instances yet. Use `architect dev` to create one.
architect-local dev:list
There are no active dev instances yet. Use `architect dev` to create one.
```

### CODE REVIEW

File: src/commands/dev/list.ts [hunk 1]
```diff
@@ -59,11 +59,7 @@ export default class DevList extends BaseCommand {
         };
       }
     }
-    if (!Object.keys(output).length) {
```
Comment (tjhiggins) [inline src/commands/dev/list.ts:62]: Talked offline, but keep the friendly error msg unless they use format json

**Review by tjhiggins — CHANGES_REQUESTED** [review]

File: src/commands/dev/list.ts (hunk 1 above)
Comment (TylerAldrich) [reply src/commands/dev/list.ts:62]: that's what this is doing I believe - this is within the `outputJSON` function and the other table output still has the error msg

Comment (mueschm) [reply src/commands/dev/list.ts:62]: This does do that. The output table function still produces the message. Only the output json function had this removed. Unless you mean something else.

Comment (devinsag) [reply src/commands/dev/list.ts:62]: Tested changes for sanity check. 

```
architect-local dev:list --format JSON
{}
architect-local dev:list --format json
{}
architect-local dev:list --format table
There are no active dev instances yet. Use `architect dev` to create one.
architect-local dev:list --format TABLE
There are no active dev instances yet. Use `architect dev` to create one.
architect-local dev:list
There are no active dev instances yet. Use `architect dev` to create one.
```

Comment (devinsag) [reply src/commands/dev/list.ts:62]: This is my fault for the horrific naming of this PR. I've updated the title to be more specific and description to include testing results.

Comment (tjhiggins) [reply src/commands/dev/list.ts:62]: ah my bad - assumed there was one output flow

**Pull request merged**